docs/specs/devcontainerjson-reference.md

```diff
@@ -79,6 +79,7 @@ While `devcontainer.json` does not focus on hardware or VM provisioning, it can
 | Property | Type | Description |
 |----------|------|-------------|
 | `hostRequirements.cpus` | integer | Indicates the minimum required number of CPUs / virtual CPUs / cores. For example: `"hostRequirements": {"cpus": 2}` |
+| `hostRequirements.gpus` | integer | Indicates the minimum required number of GPU cores. For example: `"hostRequirements": {"cpus": 2}` |
```

Comment:
The example has a typo (gpus instead of cpus):
```suggestion
| `hostRequirements.gpus` | integer | Indicates the minimum required number of GPU cores. For example: `"hostRequirements": {"gpus": 2}` |
```